org-eclipse-che-jdt-ls-extension/jdt.ls.extension.api/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: jdt.ls extension API
 Bundle-SymbolicName: jdt.ls.extension.api
-Bundle-Version: 0.0.1.qualifier
+Bundle-Version: 0.0.1
```

Comment:
The "qualifier" usually ensures that each build of a bundle gets it's own name. Why remove it, but still have -SNAPSHOT below?

Reply:
on release it was failing complaining on this so it worked only if I remove it (tbh I have no idea what it does)
error was:
```
[INFO] [ERROR] Failed to execute goal org.eclipse.tycho:tycho-packaging-plugin:1.0.0:validate-version (default-validate-version) on project jdt.ls.extension.api: Maven version 0.0.1 must have -SNAPSHOT qualifier for SNAPSHOT builds -> [Help 1]
```

Reply:
btw, idk if it should be updated manually for next version should it now be set to 0.0.2 ?
I never used this plugin